Sources/GRPC/GRPCChannel/GRPCChannelBuilder.swift

```diff
@@ -273,6 +273,16 @@ extension ClientConnection.Builder {
   }
 }
 
+extension ClientConnection.Builder {
```

Comment:
This should be an extension on `ClientConnection.Builder.Secure`: it doesn't apply to non-secure connections.